packages/components/src/date-time/time/test/index.tsx

```diff
@@ -165,7 +165,7 @@ describe( 'TimePicker', () => {
 			/>
 		);
 
-		const pmButton = screen.getByText( 'PM' );
+		const pmButton = screen.getByLabelText( 'PM', { selector: 'button' } );
```

Comment:
```suggestion
		const pmButton = screen.getByRole( 'radio', { name: 'PM' } );
```

This would be the query we want to assert the correct accessibility semantics (same with the other instances).

Reply:
Updated the query.